### PR TITLE
[Doc] Adjust doc architecture and Function Mesh contents

### PR DESCRIPTION
**Pull request by Anonymitaet**
image

### CODE REVIEW

File: docs/sqs-sink.md [hunk 1]
```diff
@@ -206,41 +237,91 @@ This example explains how to create an SQS sink connector in an on-premises clus
...
+
+- [Install the Function Mesh Operator and CRD](https://functionmesh.io/docs/install-function-mesh/) into the Kubernetes cluster.
+
+- Prepare SQS service. 
+
+  For more information, see [Getting Started with Amazon SQS](https://aws.amazon.com/sqs/getting-started/).
+
+### Step
+
+1. Define the SQS sink connector with a YAML file and save it as `source-sample.yaml`.
```
Comment (Huanli-Meng) [inline docs/sqs-sink.md:258]: `sink-sample.yaml`?

**Review by Huanli-Meng — APPROVED** [review]
overall LGTM. only left two minor comments. 

And in addition, i don't know whether we should mention that we can create a connector either through a docker image or through a package. But for SN-owned or Pulsar built-in connectors, we suggest using the docker image.

**Review by Jennifer88huang — APPROVED** [review]

File: docs/sqs-sink.md (hunk 1 above)
Comment (Anonymitaet) [reply docs/sqs-sink.md:258]: yes, thanks

**Comment by Anonymitaet** [comment]
> overall LGTM. only left two minor comments.
> 
> And in addition, i don't know whether we should mention that we can create a connector either through a docker image or through a package. But for SN-owned or Pulsar built-in connectors, we suggest using the docker image.

@Huanli-Meng thanks for your suggestions.

What do you mean by "package"? Does it refer to "nar package"? If so, the contents in the red rectangle talk about creating a connector through a nar package.
image
 
@freeznet do we need to add "how to create a connector" through docker image for the contents in the red rectangle?

**Review by freeznet — APPROVED** [review]

**Pull request merged**